handbook/support/eng-support-learn-from-eachother.md

```diff
@@ -0,0 +1,93 @@
+
+## Team to team retros
+We are not quite a big enough team to have folks pair long term with one team. From time to time, however, we may want to retro with a team. We can do this in a few ways, depending on the preference of those involved:
+
+* A CSE can join the existing engineering team’s retro session and participate (great for casual collaboration, when there isn’t anything specific, and if the engineering team agrees)
+* The CS team and X engineering team can schedule a time to do a dedicated retro together on a topic, how they are working together, etc
+* A CSE and one rep from each engineering team can join in a org wide session
+
+
+## Root cause focus
+It can be hard to focus on the root cause. It can be far too easy to focus on solving for a symptom instead. CSEs know to keep this focus and how to divide the work so that we offer a quick solution and are able to plan for the deeper work to address the underlying issue.
+
+## Support rotation donut pairing
+
+The engineering doing support rotation for their team can join the #customer-support-weekly-pairing Slack channel and be paired with a CSE for a focused coffee-style chat with the following agenda:
+
+* If CSE and engineer haven’t met before: intros, otherwise, casual catch-up
+* Engineer: what’s going on on my team? What’s being built? What am I (personally) building?
+* CSE: what’s going on in support, in general? What’s going on with supporting (search|code-intel|$TEAM)? What have been my pain points (personally)?
+* (Optional): CSE or engineer: bring any topics you’d like to learn about
+
+If the donut app pairs two engineers, we can do some swapping to ensure CSE and SWE pairings.
+
+## On-demand training materials
+Whenever an engineer or a CSE has the desire/time to create training materials they feel will help CS and that can be available on-demand (written or video), it’s as easy as:
+
+1. Create it
+2. Upload it to [the enablement folder](https://drive.google.com/drive/folders/1SSOwnsX_yNFadod88AQOxmFiINDgYoRr) in the CS shared drive and/or on the handbook CS [enablement page](support-enablement.md)
+ 
+## Documentation
+We have a shared responsibility to maintain and improve docs.sourcegraph.com. Whenever a CSE identifies a gap or something that needs to be updated, we will either:
+
+* Make the edit (or create) on the actual docs page (in Github) and seek review from the appropriate engineering team
+* Explain what is missing and ask that the appropriate engineering team fill in the gaps/needs we have identified that we don’t yet know by filing a docs issue here and tagging the right team + adding the docs tag
```

Comment:
Is the word "here" supposed to link to something? Also, the sentence reads a little strange at the moment, and ends with both "and" and "+". Maybe something like this:

```suggestion
* File a docs issue [here](TODO: missing link) explaining what is missing, and tag it as `docs` as well as tagging the appropriate engineering team to fill in the gaps/needs we have identified
```

Or something like that.